02-getting_started.Rmd

```diff
@@ -16,17 +16,17 @@ These instructions cover how to install R in Windows. The tools required to buil
 
 The `r-devel` is the next minor or eventually major release development version of R. Mostly, bug fixes are introduced in `r-patched`, while `r-devel` is for introducing new features.
 
-## Installing R
+## Installing R {#installR}
 
 1. The binary builds of R for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/). Along with the link to the latest stable release, this page also contains links to the binary builds of r-patched and r-devel. 
 
 2. Click on the download links to download an executable installer.
 
-3. Select the language while installing, read the public license information, and select the destination location to start the installation. You will be prompted to select components at this stage: `User installation`, `32-bit User installation`, `64-bit User installation`, or `Custom installation`. The default option may be opted for the questions from this step onwards to complete the installation. 
+3. Select the language while installing, read the public license information, and select destination location to the start the installation. You will be prompted to select components at this stage: `User installation`, `32-bit User installation`, `64-bit User installation`, or `Custom installation`. The default option may be opted for the questions from this step onwards to complete the installation. 

```

Comment:
redundant the; maybe "select the destination location to start the installation" instead

Reply:
"option may be opted" is also a bit redundant; how about "the default option may be chosen for the questions..."